### PR TITLE
adding styling for index.html

### PR DESCRIPTION
**Pull request by ghost**
styled the h1, h2 tags, changing color to red, aligning to the center and underlining, given p tags light gray black ground

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -19,4 +31,4 @@ <h2>lorem</h2>
 cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
 proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
 </body>
-</html>
```
Comment (benjamin-markintosh) [inline index.html:22]: change h1 color to blue

Comment (ghost) [reply index.html:22]: ok. lemme work on it

Comment (benjamin-markintosh) [inline index.html:22]: should it be having two html tags

Comment (ghost) [reply index.html:22]: made the neccesary changes.. please review

**Review by benjamin-markintosh — APPROVED** [review]

**Pull request merged**